azafea/event_processors/metrics/events/__init__.py

```diff
@@ -533,6 +575,22 @@ def _get_fields_from_payload(payload: GLib.Variant) -> Dict[str, Any]:
 
 # -- Sequence events ----------------------------------------------------------
 
+class HackClubhouseQuest(SequenceEvent):
+    __tablename__ = 'hack_clubhouse_quest'
+    __event_uuid__ = '50aebb1b-7a93-4caf-8698-3a601a0fc0f6'
+    __payload_type__ = '(bas)'
+
+    complete = Column(Boolean, nullable=False)
+    pathways = Column(ARRAY(Unicode, dimensions=1), nullable=False)
+
+    @staticmethod
+    def _get_fields_from_payload(payload: GLib.Variant) -> Dict[str, Any]:
+        return {
+            'complete': payload.get_child_value(0).get_boolean(),
+            'pathways': get_strings(payload.get_child_value(1)),
+        }
```

Comment:
As it is, only the start payload will be parsed. The stop payload will be completely discarded.

Are you ok with that?

If not, then I suggest we don't implement this event at all until Azafea grows the capacity to handle sequence events better.

Reply:
Okay, I've removed the QUEST event completely so we can add in the future when it's fully supported.